soepy/python/simulate/simulate_auxiliary.py

```diff
@@ -77,12 +87,12 @@ def pyth_simulate(model_params, states, indexer, emaxs, covariates):
         flow_utilities[:, 0] = (
             model_params.benefits ** model_params.mu
             / model_params.mu
-            * nonconsumption_utilities[0]
+            * current_non_consumption_utilities[:, 0]
```

Comment:
Very minor of course, can we use `:1` for the selection to align with Line 95?

Reply:
Done